libs/SalesforceSDKCore/SalesforceSDKCore/Classes/UserAccount/SFSDKAuthErrorManager.m

```diff
@@ -66,6 +68,20 @@ - (SFAuthErrorHandlerList *)populateDefaultAuthErrorHandlerList
     __weak typeof(self) weakSelf = self;
     SFAuthErrorHandlerList *authHandlerList = [[SFAuthErrorHandlerList alloc] init];
     
+    
+    // User agent disabled handler
+    self.userAgentErrorHandler =
+        [[SFAuthErrorHandler alloc] initWithName:kSFUserAgentErrorHandler authSessionBlock:^BOOL(NSError *error, SFSDKAuthSession *authSession, NSDictionary *options) {
+            if ([error.localizedDescription containsString:@"user-agent flow has been disabled"]) {
```

Comment:
even though it's in "localizedDescription", it looks like it's always english (even if the device, org language and user language are not)